### PR TITLE
ES-849698 - Add UG to compare two DOCX in Java - Development

### PR DESCRIPTION
**Pull request by KonduruKeerthi**
Prepared UG content to compare two word documents in Java

### CODE REVIEW

File: java-file-formats/word-library/Word-document/Compare-Word-documents.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+//Save the word document.
+originalDocument.save("Sample.docx");
+
+//Close the word document.
+originalDocument.close();
+{% endhighlight %}
+{% endtabs %}
+
+![Compare Word documents in Java](../WorkingwithWordDocument_images/Compare-Word-documents-without-author.png)
```
Comment (MathanKumarVaradhaRajaPerumal) [inline java-file-formats/word-library/Word-document/Compare-Word-documents.md:44]: Change images with revision in ballons

File: java-file-formats/word-library/Word-document/Compare-Word-documents.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+You can download a complete working sample from GitHub.
```
Comment (MathanKumarVaradhaRajaPerumal) [inline java-file-formats/word-library/Word-document/Compare-Word-documents.md:46]: please check and remove

**Review by MathanKumarVaradhaRajaPerumal — CHANGES_REQUESTED** [review]

File: java-file-formats/word-library/Word-document/Compare-Word-documents.md [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+* Insertions
+* Deletions
+* Formatting
+
+N> 1. DocIO performs word level comparison while comparing two Word documents. In this scenario, if a single character in a word is changed, the entire word will be highlighted as changed.
+N> 2. Comparing two Word documents supported in DOCX format only.
+
+## Compare two Word documents 
+
+Compare the existing Word documents or documents created from scratch using the ``Compare`` method in ``WordDocument`` class.
```
[the remark this answers is not in the document]
Comment (KonduruKeerthi) [reply java-file-formats/word-library/Word-document/Compare-Word-documents.md]: Modified as suggested

File: java-file-formats/word-library/Word-document/Compare-Word-documents.md [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+WordDocument revisedDocument = new WordDocument("RevisedDocument.docx", FormatType.Docx);
+
+//Compare the original document with the revised document.
+originalDocument.compare(revisedDocument);
+
+//Save the word document.
+originalDocument.save("Sample.docx");
+
+//Close the word document.
+originalDocument.close();
```
[the remark this answers is not in the document]
Comment (KonduruKeerthi) [reply java-file-formats/word-library/Word-document/Compare-Word-documents.md]: Modified as suggested

File: java-file-formats/word-library/Word-document/Compare-Word-documents.md (hunk 1 above)
Comment (KonduruKeerthi) [reply java-file-formats/word-library/Word-document/Compare-Word-documents.md]: Modified as suggested

File: java-file-formats/word-library/Word-document/Compare-Word-documents.md (hunk 2 above)
Comment (KonduruKeerthi) [reply java-file-formats/word-library/Word-document/Compare-Word-documents.md]: remove the line

**Review by MathanKumarVaradhaRajaPerumal — APPROVED** [review]

**Review by MohanaselvamJothi — APPROVED** [review]

**Pull request merged**